README.md

```diff
@@ -709,6 +711,17 @@ p := script.File("test.txt").Replace("old", "new")
 p := script.File("test.txt").ReplaceRegexp(regexp.MustCompile("Gol[a-z]{1}ng"), "Go")
 ```
 
+## SHA256Sum
+`SHA256Sum()` reads a list of path names from the pipe, one per line, and returns a pipe which contains the SHA-256 checksum of each pathname.
+If a line is empty, SHA256Sum will set an error to the pipe.
+
+Examples:
+
+| Input                             | `SHA256Sum` output                                                                                                                                                                                             |
+| --------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `testdata/sha256Sum.input.txt`    | `1870478d23b0b4db37735d917f4f0ff9393dd3e52d8b0efa852ab85536ddad8e`                                                                                                                                             |
+| `testdata/multiple_files/`        | `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855`<br>`e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855`<br>`e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` |
```

Comment:
I don't think this is right. If the line is a path to a directory, `SHA256Sum` will return an error, won't it? The input here is actually the set of three pathnames which represent the files in `multiple_files`.

Reply:
Yes you are right, if this is a folder I will have an error, I will correct this example.